src/main/java/org/itsallcode/openfasttrace/maven/TraceMojo.java

```diff
@@ -199,11 +198,39 @@ private ImportSettings createImportSettings()
 
     private List<Path> getSourcePaths()
     {
-        return Stream
+        return getSourcePathOfProject(this.project);
+    }
+
+    private Path getPomOfSubModule(String moduleName)
+    {
+        return this.project.getBasedir().toPath().resolve(moduleName).resolve("pom.xml");
```

Comment:
Will this work with nested projects, e.g. when the module is located in `<basedir>/plugins/moduleA`?

Reply:
Added an integration test, works ✔️